allauth/socialaccount/providers/fxa/provider.py

```diff
@@ -16,7 +51,10 @@ class FirefoxAccountsProvider(OAuth2Provider):
     account_class = FirefoxAccountsAccount
 
     def get_default_scope(self):
-        return ['profile']
+        if FXA_ON_WHITELIST:
```

Comment:
:+1: for handling the `FXA_ON_WHITELIST` setting. That tripped me up for a while when I made my first FxA oauth client.